Library/.rubocop.yml

```diff
@@ -51,6 +51,12 @@ FormulaAuditStrict:
 Homebrew:
   Enabled: true
 
+Homebrew/Blank:
+  Exclude:
+    # Core extensions are not available here:
+    - "Homebrew/rubocops/**/*"
+    - "Homebrew/startup/bootsnap.rb"
```

Comment:
Hmm, my gut instinct is that is a weird scope for a single cop (use a method one way in these paths, and not at all in these others). Possibly we could take inspiration from [Lint/DeprecatedClassMethods](https://docs.rubocop.org/rubocop/cops_lint.html#lintdeprecatedclassmethods), use a version of it to ban `blank?`, and scope the cop to these files.

I do think its worth taking a step back and thinking about whether we should just require `extend/blank` in these paths though. It seems like less trouble than maintaining separate guard rails based on paths, especially given that workspaces are generally poorly contemplated in Ruby and associated tooling (e.g. sorbet) generally.

Reply:
It turns out we already rely on `extend/blank` in our rubocops, because it was transitively available via `rubocop-rails`: https://github.com/Homebrew/brew/blob/8e9d294/Library/Homebrew/rubocops/shell_commands.rb#L123

Mind if I explicitly require it in our rubocops for now? We can circle back and refactor it away later, but i think its reasonable to maintain the status quo in this PR.